### PR TITLE
Allow the customization of the email template for reset-password message

### PR DESCRIPTION
**Pull request by otherpirate**
Hi @andrewsmedina, 

Could you please review this pull request.

Should I update the Tsuru **documentation**?
Now, we have two more configurations: 
1. `email-reset-password-template-file`: path/to/file/template.txt
2. `email-reset-password-successfully-template-file`: path/to/file/template.txt

Issue: #1164

### CODE REVIEW

**Comment by guestisp** [comment]
Cool. I think you should include docs changes in this PR so that when merged, docs are updated automatically.

File: auth/native/data.go [hunk 1]
```diff
@@ -4,9 +4,24 @@
...
 
-var resetEmailData = template.Must(template.New("reset").Parse(`Subject: [tsuru] Password reset process
+	"github.com/tsuru/config"
+)
+
+
+var passwordChars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890_@#$%^&*()~[]{}?=-+,.<>:;`"
+
+func getEmailResetPasswordTemplate() (*template.Template) {
+	templateFile, _ := config.GetString("email-reset-password-template-file")
```
Comment (andrewsmedina) [inline auth/native/data.go]: The `error` should not be ignored here. You should change this function to return `Template` and `error`:

```
func getEmailResetPasswordTemplate() (*template.Template, error) {
```

File: auth/native/data.go [hunk 2]
```diff
@@ -15,8 +30,17 @@ need to use the following token to finish this process:
 {{.Token}}
 
 If you think this is email is wrong, just ignore it.`))
-
-var passwordResetConfirm = template.Must(template.New("reset").Parse(`Subject: [tsuru] Password successfully reset
+}
+
+func getEmailResetPasswordSucessfullyTemplate() (*template.Template) {
```
Comment (andrewsmedina) [inline auth/native/data.go]: The `error` should not be ignored here. You should change this function to return `Template` and `error`.

File: auth/native/data.go [hunk 3]
```diff
@@ -15,8 +30,17 @@ need to use the following token to finish this process:
...
+	templateFile, _ := config.GetString("email-reset-password-successfully-template-file")
```
Comment (andrewsmedina) [inline auth/native/data.go]: This `config` parameter is to long. I believe that `reset-password` and `reset-password-sucessfully` is a better option. Another good option is to use namepscaes

**Comment by andrestc** [comment]
Hi @otherpirate, thanks for contributing. In addition to @andrewsmedina`s notes, can you also run check the formating with gofmt and goimports? Some checks failed on our CI.

I'm also +1 on adding the documentation to this PR.

**Comment by otherpirate** [comment]
Hi, @andrewsmedina 
I did the changes, can you review please?

**Comment by otherpirate** [comment]
Hi, @andrestc 
I checked the code with gofmt, no more problems

File: auth/native/data_test.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
+package native
+
+import (
+	"bytes"
+	"strings"
+
+	"gopkg.in/check.v1"
+
```
Comment (andrewsmedina) [inline auth/native/data_test.go]: you should remove this empty line

Comment (andrewsmedina) [inline auth/native/data_test.go]: and fix the order of imports

**Comment by otherpirate** [comment]
@andrewsmedina 
I sorted the imports by alphabetical order
Can you review again?

About Travis CI check, the problem happened in [healer_node_test.go:855](https://travis-ci.org/tsuru/tsuru/jobs/170513107) just to Go 1.6.3

**Comment by andrewsmedina** [comment]
@otherpirate thank you!

**Pull request merged**